### PR TITLE
Deleted redundant links for scRNAseq, scATACseq & LCMS

### PR DESCRIPTION
**Pull request by cebriggs7135**
All RNAseq links point to the same doc with both bulk and all sc/sn varieties of RNAseq.
All ATACseq links point to the same doc with both bulk and all sc/sn varieties of ATACseq.
All LC-MS links point to the same doc with all varieties of LC-MS

Therefore, delete all but one of each assay type and name them, respectively:
RNAseq
ATACseq
LC-MS.

### CODE REVIEW

File: context/app/markdown/docs/metadata.md [hunk 1]
```diff
@@ -36,43 +36,13 @@ Donor Specification
 - [PAS (histological stain)](/docs/assay/pas)
   - [Spec](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/stained)
   - Contact: [Jeffrey Spraggins](mailto:jeff.spraggins@Vanderbilt.Edu)
-- [RNAseq (bulk)](/docs/assay/rnaseq)
+- [RNAseq](/docs/assay/rnaseq)
   - [Spec](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/bulkrnaseq)
   - Contact: [Stephanie Nevins](mailto:snevins@stanford.edu)
-- [sciATAseq](/docs/assay/atacseq)
-  - [Spec](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/scatacseq)
```
Comment (mccalluc) [inline context/app/markdown/docs/metadata.md:43]: Note that under ATACseq above, you have
```
  - [Spec](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/bulkatacseq)
  - Contact: [Blue Lake](mailto:b1lake@eng.ucsd.edu)
```
which is different than what you're deleting here: You're not just removing redundant information.

Comment (cebriggs7135) [reply context/app/markdown/docs/metadata.md:43]: I have placed the contacts into the relevant sections of the document. There's one document for all RNAseq. There's one document for all ATACseq. There's one document for all LC-MS.
All of the RNAseq links take you to the same RNAseq doc. Likewise for ATACseq and LC-MS. They (Peter & Melissa??) created a link for each subtype of RNAseq. Likewise for RNAseq and LC-MS. The single document for RNAseq contains all of the descriptions for all of the RNAseq subtypes.

Comment (cebriggs7135) [reply context/app/markdown/docs/metadata.md:43]: I have to correct that. I put the contact links into the ATAcseq doc but have to fix that one. Will review.

Comment (mccalluc) [reply context/app/markdown/docs/metadata.md:43]: No worries; ping me again when it's ready to go.

**Comment by cebriggs7135** [comment]
@mccalluc There was one incorrect contact, which you noted above. I fixed that. I had already put the scATACseq and scRNAseq Github links into the RNAseq & ATACseq docs. I also checked that I put the correct contacts for each of the deleted links into the appropriate sc sections of the RNAseq and ATACseq docs. So all of the info under those links has been put into the appropriate subsections of the RNAseq and the ATACseq documents, respectively. 
There's just one LC-MS metadata doc in Github so that's fine. I put the contacts for each LC-MS type into the LC-MS doc as well so that's all fine as well. 
Let me know if you detect any other problems.

**Pull request merged**